completions.go

```diff
@@ -401,8 +401,9 @@ func (c *Command) getCompletions(args []string) (*Command, []string, ShellCompDi
 			doCompleteFlags := func(flag *pflag.Flag) {
 				if !flag.Changed ||
 					strings.Contains(flag.Value.Type(), "Slice") ||
-					strings.Contains(flag.Value.Type(), "Array") {
-					// If the flag is not already present, or if it can be specified multiple times (Array or Slice)
+					strings.Contains(flag.Value.Type(), "Array") ||
+					strings.Contains(flag.Value.Type(), "To") {
+					// If the flag is not already present, or if it can be specified multiple times (Array, Slice, or To)
```

Comment:
Instead of using “To” could we use “stringTo” to lower the risk of mis-detecting this case?
Specifically there is a `golangflag` type which possibly has a `Type()` we are not expecting (I’m not familiar with that typé of flash).

Reply:
Yeah, that's a better idea. Thanks for the feedback!

Reply:
I also went ahead and changed `strings.Contains()` to `strings.HasPrefix()`